force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +480,17 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
+        Decimal sum = 0.0;
+        for (Integer i = 0; i < moneyInWallet.size(); i++) {
 
-        return null; // Replace null with the variable you used to store the result
+            sum += moneyInWallet[i];
+            if (sum > 40.0) {
```

Comment:
Try using a variable for the 40 so its not a magic number
https://en.wikipedia.org/wiki/Magic_number_(programming)